src/modules/chats/components/newChat/useNewChat.tsx

```diff
@@ -12,15 +13,15 @@ export const useNewChat = () => {
         const items = res.items.filter(i => i.mutual && i.following);
         cb(
           items.map(i => {
-            const img = i.identity_meta.image || i.identity_meta.avatar || '';
-            const type = i.identity_type;
+            const { profileImage, name, type } = getIdentityMeta(i);
+            const img = profileImage;
```

Comment:
why we need a new variable and can't we use `profileImage` directly?

Reply:
No need to it. I just wanted not change the rest of the code :) I fixed it anyway